openshift-knative-operator/cmd/operator/main.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"knative.dev/operator/pkg/apis/operator"
+	operatorv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
```

Comment:
This and the conversion logic forces us to bump to a previous version. Unfortunately 1.6.x completely removes that version.